src/components/media/controls/controls.tsx

```diff
@@ -0,0 +1,299 @@
+/* eslint-disable no-magic-numbers */
+import './controls.css'
+
+import Forward10RoundedIcon from '@mui/icons-material/Forward10Rounded'
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown'
+import KeyboardArrowUpIcon from '@mui/icons-material/KeyboardArrowUp'
+import PauseCircleFilledIcon from '@mui/icons-material/PauseCircleFilled'
+import PlayCircleFilledIcon from '@mui/icons-material/PlayCircleFilled'
+import Replay10RoundedIcon from '@mui/icons-material/Replay10Rounded'
+import VolumeOffRoundedIcon from '@mui/icons-material/VolumeOffRounded'
+import VolumeUpRoundedIcon from '@mui/icons-material/VolumeUpRounded'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import IconButton from '@mui/material/IconButton'
+import Slider from '@mui/material/Slider'
+import Typography from '@mui/material/Typography'
+import React, { useEffect, useState } from 'react'
+
+import { formatDurationTime } from '../../helper'
+
+interface MediaIconButtonProps {
+  onClick: () => void
+  ariaLabel: string
+  icon: React.ReactNode
+  className?: string
+  dataCy?: string
+}
+
+export function MediaIconButton(props: MediaIconButtonProps) {
+  return (
+    <IconButton
+      onClick={props.onClick}
+      aria-label={props.ariaLabel}
+      className={props.className}
+      data-cy={props.dataCy}
+    >
+      {props.icon}
+    </IconButton>
+  )
+}
```

Comment:
I've been thinking that tweaking this to use material-symbols will reduce redundant props and make it easier to maintaining uniformity
```suggestion
export function MediaIconButton(props: MediaIconButtonProps) {

  const controls = {
    play: { symbol: 'play_circle', label: 'play' },
    pause: { symbol: 'pause_circle', label: 'pause' },
    forward: { symbol: 'forward_10', label: 'forward ten seconds' },
    replay: { symbol: 'replay_10', label: 'replay ten seconds' },
    volumeUp: { symbol: 'volume_up', label: 'volume up' },
    volumeOff: { symbol: 'volume_off', label: 'volume off' },
  }

  return (
    <IconButton
      onClick={props.onClick}
      aria-label={controls[props.action].label}
      className={props.className}
      data-cy={`${props.action}-button`}
    >
      <Icon fontSize={props.fontSize? props.fontSize : 'medium'} color="primary">
        <span className="material-symbols-outlined">{controls[props.action].symbol}</span>
      </Icon>
    </IconButton>
  )
}
```